lib/fastlane/plugin/ddg_apple_automation/helper/ddg_apple_automation_helper.rb

```diff
@@ -132,6 +133,61 @@ def self.prepare_release_branch(platform, version, other_action)
         return release_branch_name, new_version
       end
 
+      def self.prepare_hotfix_branch(platform, version, other_action, options)
+        UI.user_error!("You must provide a version you want to hotfix.") unless version
+        source_version = validate_version_exists(version)
+        new_version = validate_hotfix_version(source_version)
+        release_branch_name = create_hotfix_branch(platform, source_version, new_version)
+        increment_build_number(platform, options, other_action)
+        Helper::GitHubActionsHelper.set_output("release_branch_name", release_branch_name)
+
+        return release_branch_name, new_version
+      end
+
+      def self.create_hotfix_branch(platform, source_version, new_version)
+        branch_name = "#{HOTFIX_BRANCH}/#{new_version}"
+        UI.message("Creating new hotfix release branch for #{new_version}")
+
+        existing_branch = Actions.sh("git", "branch", "--list", branch_name).strip
+        UI.abort_with_message!("Branch #{branch_name} already exists in this repository. Aborting.") unless existing_branch.empty?
+
+        if Helper.is_ci?
+          sha = Actions.sh("git", "rev-parse", "#{source_version}^{}").strip
+          repo = Helper::GitHelper.repo_name(platform)
+          Actions.sh("gh", "api", "--method", "POST", "/repos/#{repo}/git/refs", "-f", "ref=refs/heads/#{branch_name}", "-f", "sha=#{sha}")
```

Comment:
I'd rather not use `gh` CLI tool directly in fastlane:
* It requires GitHub token to be set (which needs to be done manually in the GHA step environment)
* some of our actions (including start_new_release), already take GH token as parameter so it would require extra setup to have this work.

We have Octokit to access GitHub API. Can we create a helper function in git_helper to create a branch? Similar to existing `merge_branch` and `delete_branch`? Maybe called `create_branch`.

Alternatively, may be it's worth considering dropping the entire `is_ci` thing and just sticking to git fetch & checkout & push?